test-clients/src/main/resource/contract/contracts/ExchangeRatePrecompile/RateAware.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity >=0.5.0 <0.9.0;
+
+abstract contract RateAware {
+    uint64 constant TINY_PARTS_PER_WHOLE = uint64(100_000_000);
+    bytes1 constant TO_TINYBARS = bytes1(0xbb);
+    bytes1 constant TO_TINYCENTS = bytes1(0xcc);
+    address constant precompileAddress = address(0x21D4);
+
+    function toTinybars(int64 tinycents) 
```

Comment:
This reads well in the definition, but I think `tinycentsToTinybars` would read better where it is used.

Reply:
Changed names to `tinycentsToTinybars` and `tinybarsToTinycents` in [new `IExchangeRate` interface](https://github.com/hashgraph/hedera-services/blob/03301-exchange-rate-system-contract/test-clients/src/main/resource/contract/contracts/ExchangeRatePrecompile/IExchangeRate.sol#L4).